packages/sdk/governance/src/proposals.ts

```diff
@@ -206,7 +206,10 @@ export class ProposalBuilder {
 
   fromJsonTx = async (tx: ProposalTransactionJSON): Promise<ProposalTransaction> => {
     // handle sending value to unregistered contracts
-    if (!RegisteredContracts.includes(tx.contract)) {
+    if (
+      !RegisteredContracts.includes(tx.contract.toString().replace('Proxy', '') as CeloContract) &&
```

Comment:
nit: Just to make sure no weird edge case fall is encountered, can the replace function be applied to only the last 5 chars of the string?